spec/csv_row_model/integrations/dynamic_columns_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'spec_helper'
+
+describe 'Dynamic columns scenario' do
+
+
+  context 'import' do
+
+    class ImportDynamicColumnModel < DynamicColumnModel
+      include CsvRowModel::Import
+
+      def self.skills_request
```

Comment:
We need to access to `context` for doing something like : 
`Project.where(id: context.project_ids)`

same for headers
`# context.project_ids # Provided by dashboard selection`